rules/renaming/src/Rector/StaticCall/RenameStaticMethodRector.php

```diff
@@ -96,7 +96,7 @@ private function rename(StaticCall $staticCall, RenameStaticMethod $renameStatic
         $staticCall->name = new Identifier($renameStaticMethod->getNewMethod());
 
         if ($renameStaticMethod->hasClassChanged()) {
-            $staticCall->class = new Name($renameStaticMethod->getNewClass());
+            $staticCall->class = new Name('\\' . $renameStaticMethod->getNewClass());
```

Comment:
The `FullyQualified` has the same effect, just internally adds `\\` to it's node.
It's also better to use it, as then further analysis knows its FQN

```suggestion
            $staticCall->class = new FullyQualified($renameStaticMethod->getNewClass());
```

Reply:
implemented.